tests/unit_tests/Commands/Site/Upstream/SetCommandTest.php

```diff
@@ -85,16 +88,20 @@ protected function setup()
         $this->site_upstream->id = 'Site upstream ID';
         $this->upstream_data = ['framework' => 'Framework', 'id' => 'upstream_id', 'label' => 'Upstream Name',];
 
+        parent::setUp();
```

Comment:
This looked like an odd change, just pointing that out as it might make perfect sense to you.